core/src/main/java/org/mule/runtime/core/internal/policy/DefaultOperationPolicyProcessorFactory.java

```diff
@@ -18,19 +17,13 @@
  */
 public class DefaultOperationPolicyProcessorFactory implements OperationPolicyProcessorFactory {
 
-  private final PolicyStateHandler policyStateHandler;
-
   /**
    * Creates a new {@link Processor} from an operation {@link Policy}.
-   *
-   * @param policyStateHandler the state handler to use for keeping track of the policy chain event modifications.
    */
-  public DefaultOperationPolicyProcessorFactory(PolicyStateHandler policyStateHandler) {
-    this.policyStateHandler = policyStateHandler;
-  }
+  public DefaultOperationPolicyProcessorFactory() {}
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. ([squid:S1186](https://sonarqube.kbuild.msap.io/coding_rules#rule_key=squid%3AS1186))

[See it in SonarQube](https://sonarqube.kbuild.msap.io/project/issues?id=mule-runtime-mule&pullRequest=7629&issues=AWiGlnW3otIXnmsUoniS&open=AWiGlnW3otIXnmsUoniS)

[ISSUE_KEY]: AWiGlnW3otIXnmsUoniS